libs/brain/collapsible/src/lib/brn-collapsible-content.ts

```diff
@@ -1,40 +1,24 @@
-import { isPlatformServer } from '@angular/common';
-import {
-	ChangeDetectionStrategy,
-	Component,
-	ElementRef,
-	type OnInit,
-	PLATFORM_ID,
-	effect,
-	inject,
-	input,
-	signal,
-	untracked,
-} from '@angular/core';
-import { BrnCollapsible } from './brn-collapsible';
+import { Directive, ElementRef, afterNextRender, effect, inject, input, signal, untracked } from '@angular/core';
+import { injectBrnCollapsible } from './brn-collapsible-token';
 
-@Component({
-	selector: 'brn-collapsible-content',
+@Directive({
+	selector: '[brnCollapsibleContent],brn-collapsible-content',
 	host: {
 		'[hidden]': '!_collapsible?.expanded()',
```

Comment:
since you're already visiting this component, i suggest tackling this issue: 

https://github.com/spartan-ng/spartan/issues/883

it makes the collapsible primitive impossible to animate properly